oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -18,14 +18,11 @@
  */
 package oshi.software.os;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
 import org.junit.Test;
-
 import oshi.SystemInfo;
-import oshi.software.common.AbstractProcess;
+import oshi.software.os.windows.WindowsProcess;

```

Comment:
Can't use a platform-specific class in a test that will run on any OS.  Running this test on Linux results in the below:

> testGlobalMemoryCopy(oshi.hardware.GlobalMemoryTest)  Time elapsed: 0.024 sec  <<< ERROR!
> 
> java.lang.UnsatisfiedLinkError: Unable to load library 'Psapi': Native library (linux-x86-64/libPsapi.so) not found in resource path